homework/index.js

```diff
@@ -30,13 +30,93 @@
     return elem;
   }
 
+  function renderContributors(contributors, contributorsContainer) {
+    createAndAppend('p', contributorsContainer, { text: 'Contributors', id: 'contributors' });
+    contributors.forEach(contributor => {
+      const contributorsDetailsDiv = createAndAppend('div', contributorsContainer, {
+        class: 'contributor-details',
+      });
+      createAndAppend('img', contributorsDetailsDiv, {
+        src: contributor.avatar_url,
+        alt: contributor.login,
+        class: 'image',
+      });
+      createAndAppend('p', contributorsDetailsDiv, {
+        text: contributor.login,
+        href: contributor.html_url,
```

Comment:
An `href`` attribute is not allowed on a `p` element. This is the full error report from the HTML Validator on the HTML your app generates (the line number refers to the generated HTML and have no relation to this JavaScript file).

```
Error: Attribute “href” not allowed on element “p” at this point.
    From line 16, column 2454; to line 16, column 2530
    Error: Attribute “target” not allowed on element “p” at this point. 
    From line 16, column 2454; to line 16, column 2530
    Error: Attribute “href” not allowed on element “p” at this point.   
    From line 16, column 2709; to line 16, column 2788
    Error: Attribute “target” not allowed on element “p” at this point. 
    From line 16, column 2709; to line 16, column 2788
```

Reply:
Hi @jim, 
Thanks for the feedback. I have been so engaged with something else but will keep you updated once the changes are made. Thanks!

Reply:
Hi @jim,
The changes are made except for the responsiveness which will be done in week2.